packages/engine/Source/Scene/ArcGisMapServerImageryProvider.js

```diff
@@ -357,10 +429,26 @@ function ArcGisMapServerImageryProvider(options) {
       }
 
       that._useTiles = true;
+    } else {
+      const message = `The service does not have any Pre Cached Tiles or the required tileInfo resources and is not supported.`;
```

Comment:
This is technically a breaking change and will need to be noted as such in `CHANGES.md`